app/views/snps/show.html.erb

```diff
@@ -59,6 +63,22 @@
       We don't have enough users yet.
     <%end%>
   </div>
+
+  <div class="span3 columns">
+    <center><h6><a href="#" rel="tooltip" data-html="true" title="Phenotypes possibly associated with this SNP.">Phenotypes</a></h6></center>
+    <% if @snp.phenotypes.count %>
+      <table id="phenotype_table" class="table table-striped">
+        <% @ordered_phenotype_snps.each do |p| %>
+          <tr>
+            <td> <%= Phenotype.find(p.phenotype_id).characteristic %> </td>
```

Comment:
Here is the documentation, if you are interested:
http://api.rubyonrails.org/classes/ActiveRecord/QueryMethods.html#method-i-includes